Source/p_mobj.c

```diff
@@ -532,6 +532,9 @@ static void P_ZMovement (mobj_t* mo)
 
 		mo->player->deltaviewheight = mo->momz>>3;
 		S_StartSound (mo, sfx_oof);
+		// [crispy] center view if not using permanent mouselook
+//		if (!crispy->mouselook)
```

Comment:
keep it or not?

Reply:
Autocenter after releasing the "mouselook" button? I think we need it.